### PR TITLE
Removed the send all button from the send screen

### PR DESCRIPTION
**Pull request by juanky201271**
The best idea is to remove the 'send all' button, It's not very useful and very dangerous. Thank you @Gunnar-Stunnar for doing that.

### CODE REVIEW

File: components/SendScreen.tsx [hunk 1]
```diff
@@ -573,11 +573,11 @@ const SendScreen: React.FunctionComponent<SendScreenProps> = ({
               <View style={{display: 'flex', flexDirection: 'column'}}>
                 <View style={{display: 'flex', flexDirection: 'row', marginTop: 10}}>
                   <FadeText>Spendable: ᙇ {Utils.maxPrecisionTrimmed(getMaxAmount())} </FadeText>
-                  <ClickableText
+                  {/* <ClickableText
```
Comment (zancas) [inline components/SendScreen.tsx:576]: What happens if we simply delete line 576 and 580?

What happens if we delete lines 576 - 580?

All other things being equal, I vote that we delete lines 576-580.

Comment (juanky201271) [reply components/SendScreen.tsx:576]: Of course, It's cleaner...

Comment (zancas) [reply components/SendScreen.tsx:576]: I would do it myself, but I am not yet running the tests.  I intend to acquire that capability soon.

Comment (zancas) [reply components/SendScreen.tsx:576]: Alright...  I will do this soon, if it's still outstanding.

Comment (juanky201271) [reply components/SendScreen.tsx]: I did.

**Comment by juanky201271** [comment]
This chunk of code is been removed.

**Comment by zancas** [comment]
Woot!

**Pull request merged**